### PR TITLE
Adjusted the exchange box size to its content

### LINKED ISSUE #12: Adjust exchangeBox size dynamically
Resize the box based on its content instead of having a fixed size.

### PR DESCRIPTION
**Pull request by landao2607**

### CODE REVIEW

**Review by daotalabs — DISMISSED** [review]
nice nice

File: css/styles.css [hunk 1]
```diff
@@ -17,10 +17,10 @@
...
 	z-index: 10000;
 	/* for the tooltip box */
-	width: 120px;
-	bottom: 100%;
-	height: 50px;
-	margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
+	width: auto;
+	bottom: auto;
+	height: auto;
+	margin-left: auto;
```
Comment (vietdao204) [inline css/styles.css:23]: may not need this `margin-left`

File: css/styles.css [hunk 2]
```diff
@@ -17,10 +17,10 @@
 	position: absolute;
 	z-index: 10000;
 	/* for the tooltip box */
-	width: 120px;
-	bottom: 100%;
-	height: 50px;
-	margin-left: -60px; /* Use half of the width (120/2 = 60), to center the tooltip */
+	width: auto;
+	bottom: auto;
+	height: auto;
```
Comment (daotalabs) [inline css/styles.css]: `width`, `bottom` and `height` are `auto` by default so you can remove these
https://www.w3schools.com/cssref/pr_dim_width.asp

File: css/styles.css (hunk 1 above)
Comment (vietdao204) [reply css/styles.css:23]: `margin-left: -40px` makes the box looks more aligned below the cursor so let's do that

File: css/styles.css (hunk 2 above)
Comment (vietdao204) [reply css/styles.css:22]: https://www.w3schools.com/cssref/pr_dim_width.asp

File: css/styles.css (hunk 1 above)
Comment (daotalabs) [inline css/styles.css]: `margin-left: -40px` makes the box better align below the cursor so let's use that

**Review by daotalabs — APPROVED** [review]

**Pull request merged**